packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -0,0 +1,35 @@
+export const WaferColorByOptions = {
```

Comment:
We have a pattern for writing tests for each enum-like type to make sure we include the right attributes on the type (like `as const`). Could you add `tests/types.spec.ts` and write a test for each new type following the pattern of type tests for other components (e.g. [button types](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/button/tests/types.spec.ts))?

Reply:
Yes, sure, I will add :)

Reply:
I added tests for all the types